src/acl/acl.ts

```diff
@@ -373,7 +374,7 @@ export function fixIndividualCardACL (person: $rdf.NamedNode, log: Function, cal
  * This function is used by [[fixIndividualCardACL]]
  */
 export function fixIndividualACL (item: $rdf.NamedNode, subjects: Array<$rdf.NamedNode>, log: Function, callbackFunction: Function): void {
-  log = log || console.log
+  log = log || debug.log
```

Comment:
Think this could be set as default param for the log parameter, but of course that begs to ask what we should set as default value for callbackFunction. Nevertheless something we want to clean up later, to not make this PR too extensive.